src/Files.App/Utils/Cloud/Detector/GoogleDriveCloudDetector.cs

```diff
@@ -175,8 +187,8 @@ private static async Task Inspect(SqliteConnection database, string sqlCommand,
 				.RootElement.EnumerateObject()
 				.FirstOrDefault();
 
-			// A default JsonProperty struct has an "Undefined" Value#ValueKind and throws an
-			// error if you try to call EnumerateArray on its Value.
+			// A default `JsonProperty` struct has an Undefined `Value.ValueKind` and throws an
+			// error if you try to call `EnumerateArray` on its `Value`.
```

Comment:
```suggestion
			// A default "JsonProperty" struct has an undefined "Value.ValueKind" and throws an
			// error if you try to call "EnumerateArray" on its value.
```

Comments don't render like markdown, and quotes look better in most IDEs.